definitions/checks/check_for_newer_packages.rb

```diff
@@ -0,0 +1,25 @@
+class Checks::CheckForNewerPackages < ForemanMaintain::Check
+  metadata do
+    label :check_for_newer_packages
+    description 'Check for newer packages and optionally ask for confirmation if not found.'
+
+    param :packages,
+          'package names to validate',
+          :required => true
+    param :manual_confirmation_version,
+          'Version of satellite (6.9) to ask the user if they are on the latest z-stream of.',
+          :required => false
+  end
+
+  def run
+    result = ForemanMaintain.package_manager.check_update(packages: @packages, with_status: true)
```

Comment:
Maybe you can use vars like:
```suggestion
    exit_status, _ = ForemanMaintain.package_manager.check_update(packages: @packages, with_status: true)
```